<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bookadress</class>
 <widget class="QWidget" name="Bookadress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bookadress</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_secName">
       <property name="text">
        <string>Фамилия:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_secName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Name">
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_Name"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Patronymic">
       <property name="text">
        <string>Отчество:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_Patronymic"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_Adress">
       <property name="text">
        <string>Адрес:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_Adress"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_HomePhone">
       <property name="text">
        <string>Дом.тел:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_HomePhone"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_MobPhone">
       <property name="text">
        <string>Моб.тел:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_MobPhone"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_Others">
       <property name="text">
        <string>Другое:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_others"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
